Snowflake.Data/Core/SFSession.cs

```diff
@@ -72,7 +74,12 @@ static SFSession()
             AuthnRequestClientEnv clientEnv = new AuthnRequestClientEnv()
             {
                 application = System.Diagnostics.Process.GetCurrentProcess().ProcessName,
-                osVersion = System.Environment.OSVersion.VersionString
+                osVersion = System.Environment.OSVersion.VersionString,
+#if NET46
+                netRuntime = "CLR:" + Environment.Version.ToString()
+#else
+                netRuntime = System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription
+#endif

```

Comment:
what is the value `netRuntime` look like? Can you give examples? Can it be combined with `OS_VERSION`?

In case of ODBC driver, OS_VERSION consists of the windows version followed by architecture.
```
"6.1-x86_64"
```

Reply:
The initial idea of NET_RUNTIME is to distinguish .net core or .net framework. However, like I said, it is really difficult to get .net version in runtime. dotnet/corefx#17452

Reply:
Ah I remember that. So we are providing two binaries to nuget.

Reply:
Yeah, two binaries in one package.